polus-imagej-plugins/polus-imagej-image-integral-plugin/tests/version_test.py

```diff
@@ -0,0 +1,24 @@
+import unittest, json
+from pathlib import Path
+
+class VersionTest(unittest.TestCase):
+    
+    version_path = Path(__file__).parent.parent.joinpath("VERSION")
+    json_path = Path(__file__).parent.parent.joinpath("plugin.json")
+    
+    def test_plugin_manifest(self):
+        
+        # Get the plugin version
+        with open(self.version_path,'r') as file:
+            version = file.readline()
+            
+        # Load the plugin manifest
+        with open(self.json_path,'r') as file:
+            plugin_json = json.load(file)
+        
+        self.assertEqual(plugin_json['version'],version)
+        self.assertTrue(plugin_json['containerId'].endswith(version))
```

Comment:
Test is good. Would be nice to see actual functionality tested.

Reply:
I think that will be the next step in building the generation code. Using the info we have scraped from introspection to generate good unit tests. I don't even think it will be super hard, since we have all the needed information, it will just take some planning and jinja.